builds/e2e/templates/e2e-run.yaml

```diff
@@ -18,22 +20,30 @@ steps:
       $filter += '&Category!=UnstableOnArm'
     }
     
-    #filter out tests not usable on nested edge
-    if ('$(nestededge)' -eq 'true')
+    if ($test_type -eq 'nestededge_mqtt')
     {
       $filter += '&Category!=SingleNodeOnly'
```

Comment:
>$filter += '&Category!=SingleNodeOnly' [](start = 6, length = 38)

I think this needs to apply to nestededge_amqp as well

Reply:
Good catch